### PR TITLE
[AAP-84893] Regenerate lockfile: redis 7.4.1 → 8.1.0

### PR DESCRIPTION
**Pull request by tznamena**
## Description

- **What:** Regenerate the pip lockfile (`requirements/requirements.txt`) to resolve `redis` from 7.4.1 to 8.1.0.
- **Why:** The previous commit (b6d768ee, AAP-84893) added `redis>=8.0.1,<9` to `requirements.in` but deferred lockfile regeneration because DAB still had a `redis<8.0.0` version cap. That cap was removed in django-ansible-base#1075, but the lockfile was never updated. This causes the container build to fail — `pip install` sees the package metadata requiring `redis>=8.0.1` but the compiled lockfile only provides 7.4.1.
- **How:** Ran `./updater.sh run` to regenerate the lockfile. The only version change is `redis 7.4.1 → 8.1.0`; all other diff lines are cosmetic (`# via` comment path normalization by pip-compile).

## Type of Change

- [x] Bug fix (non-breaking change which fixes an issue)

## Self-Review Checklist

- [x] I have performed a self-review of my code
- [x] I have added relevant comments to complex code sections
- [x] I have updated documentation where needed
- [x] I have considered the security impact of these changes
- [x] I have considered performance implications
- [x] I have thought about error handling and edge cases
- [x] I have tested the changes in my local environment

### Prerequisites
None — this is a lockfile-only change.

### Steps to Test
1. Build the container with this version of the gateway
2. Verify the container builds successfully without `redis` version conflicts

### Expected Results
Container build completes without `Could not find a version that satisfies the requirement redis>=8.0.1` errors.

## Summary by CodeRabbit

* **Chores**
  * Updated dependency attribution metadata for `django-redis`, `python-ldap`, and Redis.
  * Upgraded Redis from version 7.4.1 to 8.1.0.

### CODE REVIEW

File: requirements/requirements.txt [hunk 1]
```diff
@@ -42,47 +42,51 @@ django==5.2.9
     #   djangorestframework
     #   drf-spectacular
     #   social-auth-app-django
+django-ansible-base[activitystream,api-documentation,authentication,feature-flags,oauth2-provider,observability,rbac,redis-client,rest-filters] @ git+https://github.com/ansible/django-ansible-base@devel
+    # via -r requirements_git.txt
```
[the remark this answers is not in the document]
Comment (tznamena) [reply requirements/requirements.txt]: This is intentional.

**Review by Dostonbek1 — APPROVED** [review]

**Review by jay-steurer — APPROVED** [review]
